packages/webdriverio/tests/__mocks__/request.js

```diff
@@ -130,6 +130,60 @@ const requestMock = jest.fn().mockImplementation((params, cb) => {
         break
     }
 
+    /**
+     * Simulate a stale element
+     */
+
+    if (params.uri.path === `/wd/hub/session/${sessionId}/element/${genericSubSubElementId}/click`) {
+        ++requestMock.retryCnt
+
+        if (requestMock.retryCnt > 1) {
+            const response = { value: null }
+            return cb(null, {
+                headers: { foo: 'bar' },
+                statusCode: 200,
+                body: response
+            }, response)
+        }
+
+        let error = new Error('element is not attached to the page document')
+        error.name = 'stale element reference';
+
+        return cb(error, {
+            headers: { foo: 'bar' },
+            statusCode: 404,
+            body: {}
+        }, {});
+    }
+
+
+    /**
+     * Simulate a stale element
+     */
+
+    if (params.uri.path === `/wd/hub/session/${sessionId}/element/${genericSubSubElementId}/click`) {
```

Comment:
This if will never be reached because there is the same condition in the if in line 137 which will return

Reply:
Good catch. Looks like i had a bad merge on my final rebase. Will fix.

Reply:
Fixed.